Hosting/NetCord.Hosting.Services/Commands/CommandResultHandler.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Extensions.Logging;
+
+using NetCord.Gateway;
+using NetCord.Services;
+using NetCord.Services.Commands;
+
+namespace NetCord.Hosting.Services.Commands;
+
+public class CommandResultHandler<TContext>(MessageFlags? onFailResultResponseFlags = null) : ICommandResultHandler<TContext> where TContext : ICommandContext
+{
+    public MessageFlags? OnFailResultResponseFlags { get; set; } = onFailResultResponseFlags;
```

Comment:
Do we actually need the property? Also the name would just be `[Mm]essageFlags` I think, this one is really long.

Reply:
Why would we not need the property? The method has to access it somehow, no?

Reply:
It can just access the constructor parameter

Reply:
No reason to make it a readonly field, the loss is so extremely minimal compared to the *potential* usecases of accessing it outside the ctor. I wanted to make the name long because it provides clarity what it is for

Reply:
> It can just access the constructor parameter

I didn't know that was possible, I see